src/service/crypto-impl.js

```diff
@@ -215,6 +215,36 @@ export class Crypto {
       data
     ));
   }
+
+  /**
+   * Returns a cryptographically random string with 128 bits of entropy if win.crypto is available.
+   * If win.crypto, is not available, returns null.
+   * @param {number} length The length of the string to return
+   * @return {string|null}
```

Comment:
Let's have this return the array or null.  You can then convert the array to a string within the doubleclick code (assumes other future consumers will want an array for other usage cases)

Reply:
done